examples/batch.ts

```diff
@@ -16,24 +17,32 @@ import {
 //   }),
 // )
 
-const tx = extrinsic({
-  sender: T.alice.address,
-  call: Utility.batchAll({
-    calls: T.users.map((pair) =>
-      Balances.transfer({
-        dest: pair.address,
-        value: 12345n,
-      })
-    ),
-  }),
-})
-  .signed(T.alice.sign)
-  .watch((ctx) => (status) => {
-    console.log(status)
-    if (C.rpc.known.TransactionStatus.isTerminal(status)) {
-      return ctx.end()
-    }
-    return
-  })
+const senderKP = Sr25519.fromSeed(crypto.getRandomValues(new Uint8Array(32)))
```

Comment:
Instead of creating a new user, could you send from one of the other default users, e.g. `T.dave`?

If not, I think this example should be temporarily ignored instead of complicating it like this.